tests/tensor/test_math_opt.py

```diff
@@ -4079,7 +4079,7 @@ def check_max_log_sum_exp(x, axis, dimshuffle_op=None):
 
         # in mode FAST_COMPILE, the optimisations don't replace the
         # MaxAndArgmax op.
-        if isinstance(node.op, MaxAndArgmax):
+        if isinstance(node.op, Max, Argmax):
```

Comment:
```suggestion
        if isinstance(node.op, (Max, Argmax)):
```
This appears to be the cause of the current CI error.